doc/modules/preprocessing.rst

```diff
@@ -574,6 +574,43 @@ When this paramenter is not None, ``handle_unknown`` must be set to
 See :ref:`dict_feature_extraction` for categorical features that are represented
 as a dict, not as scalars.
 
+.. _one_hot_encoder_infrequent_categories:
+
+Infrequent categories
+---------------------
+
+:class:`OneHotEncoder` supports creating a category for infrequent categories
+in the training data. The parameters to enable the gathering of infrequent
+categories are `min_frequency` and `max_levels`.
+
+1. `min_frequency` can be a integer greater or equal to one or a float in
+`(0.0, 1.0)`. If `min_frequency` is an integer, categories with a cardinality
+smaller than this value will be considered infrequent. If `min_frequency` is an
+float, categories with a cardinality smaller than this fraction of the
+total number of samples will be considered infrequent.
+
+2. `max_levels` can be `None` or any integer greater than one. This parameter
+sets an upper limit of the number of categories including the infrequent
+category.
+
+These parameters can be used together to filter out infrequent categories. In
```

Comment:
The following example does not use these parameters together, so this sentence seems a bit out of place.